release-tools/check_plugin.sh

```diff
@@ -0,0 +1,196 @@
+#!/bin/bash
+
+###### Information ############################################################################
+# Name:          check_plugin.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
```

Comment:
Can you please remove the Name, Maintainer and the language.

Reply:
Why?

Reply:
Not required. People can see name of the file and extension and since we own this repo we know who owns it

Reply:
It is a comment, and has this part on many other scripts.
I dont see the harm of keeping it here.
Nor did I see the harm of removing it.
I will just leave it here.